content/docs/introduction/plans.md

```diff
@@ -14,88 +14,138 @@ redirectFrom:
 updatedOn: '2024-02-27T16:17:40.463Z'
 ---
 
-Neon's plans are designed to meet different user requirements, ranging from hobby projects to enterprise-level production workloads. We also offer custom enterprise plans with volume-based discounts for large teams or database fleets. Refer to our [pricing](https://neon.tech/pricing) page for an overview of our plans and pricing. 
+Neon's plans are designed to meet different user requirements, ranging from hobby projects to enterprise-level production workloads. We also offer custom enterprise plans with volume-based discounts for large teams or database fleets. Refer to our [Pricing](https://neon.tech/pricing) page for fees and a detailed plan comparison. 
 
-This page provides additional information to help you understand our plans and find the one that's right for you.
+Neon offers four plans: 
 
-## Pricing plans
+- [Free Tier](#free-tier)
+- [Launch](#launch)
+- [Scale](#scale)
+- [Enterprise](#enterprise)
 
-Neon offers four pricing plans: [Free Tier](#free-tier), [Launch](#launch), [Scale](#scale), and [Enterprise](#enterprise).
+<Admonition type="tip" title="Plan Allowances and Extra Usage">
+Neon plans are structured around **Allowances** and **Extra usage**. Allowances are included in your plan. With Neon's paid plans, extra usage is available for a fee.
```

Comment:
Make Extra usage here a link to the extra usage page?

Reply:
done!